### ISSUE TITLE
Bug: "Display Segments in Reverse Order" sorts nulls to the top

### ISSUE THREAD

**Issue by MelGrubb** [jacobtread/obsidian-timekeep#19]
I have included Timekeep in my daily note template with three entries for common tasks I do every day. If I put the items in reverse order, though, all the ones I haven't started yet appear at the top of the list. It seems like unstarted tasks should always appear at the bottom of the list, regardless of which order I've sorted the completed tasks in.

**To Reproduce**
Use the entries below to see the problem for yourself.

**Expected behavior**
Unstarted entries should always appear at the end of the list.

**Screenshots**

**Additional context**
```json
{"entries":[{"name":"Troubleshooting","startTime":"2024-12-26T12:25:00.000Z","endTime":"2024-12-26T12:44:00.000Z","subEntries":null},{"name":"Development","startTime":"2024-12-26T12:44:00.000Z","endTime":null,"subEntries":null},{"name":"Meetings","startTime":null,"endTime":null,"subEntries":null}]}
```

**Comment by jacobtread** [comment]
This will be because that is not actually sorting. Currently, it's just a direct reverse order of the actual array itself "display segments in reverse order" as in reverse insertion order, not reverse start order so just the reverse order of how you had them in your unstarted template

I'll pop up a feature request when I've got a chance to add a proper sorting option that actually sorts thr list using the timestamps

**Comment by jacobtread** [comment]
Implemented in #21 released in latest release [1.6.0](https://github.com/jacobtread/obsidian-timekeep/releases/tag/1.6.0) should be available for you to update now

**Comment by MelGrubb** [comment]
It looks good. I'm trying it out today.
